front/src/common/BootstrapSNCF/InputGroupSNCF.tsx

```diff
@@ -88,11 +90,12 @@ export default function InputGroupSNCF({
         title={placeholder}
         placeholder={placeholder}
         onChange={(e) => handleType({ type: selected.id, value: e.target.value })}
-        value={value}
+        value={value !== undefined ? value : ''}
```

Comment:
you can use the nullish coalescent operator
```suggestion
        value={value ?? ''}
```

Reply:
I've added a new test, so I think I can't use the nullish coalescent operator now

Reply:
ok closing